suites/pacific/rgw/sanity_rgw.yaml

```diff
@@ -180,6 +180,16 @@ tests:
         config-file-name: test_swift_basic_ops.yaml
         timeout: 300
 
+  - test:
+      name: swift versioning copy tests
+      desc: restore versioned object in swift
+      polarion-id: CEPH-10646
```

Comment:
This script points to for s3 here, [CEPH-10646](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-10646) and not swift versioning test, can you take a look at it and update. but do not remove s3 test script in polarion.